### PR TITLE
LPS-130860 Capture return key event and delete pop-up from DOM after destroy portlet

### PR DESCRIPTION
**Pull request by ramonmsp**
Issue: [LPS-130860](https://issues.liferay.com/browse/LPS-130860)

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Hey @MarinhoFeliphe, the `com.liferay.portal.workflow.metrics.rest.resource.v1_0.test.InstanceResourceTest.testGetProcessInstancesPage` failed again :( . 

Hey @rodrigocunhaa, Can we know the failure rate of a test case running on CI? For example, the `com.liferay.portal.workflow.metrics.rest.resource.v1_0.test.InstanceResourceTest.testGetProcessInstancesPage`.


Thx guys

**Comment by rafaprax** [comment]
ci:stop

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/main.js [hunk 1]
```diff
@@ -230,7 +230,20 @@ AUI.add(
 				_onDestroyPortlet() {
 					var instance = this;
 
+					const scriptCellEditor = document.querySelector(
+						'.script-cell-editor'
+					);
+
 					instance.destroy(true);
+
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/main.js:238]: Hey @ramonmsp, move `instance.destroy(true);` to the end of function. pls.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rodrigocunhaa** [comment]
Hey @rafaprax , according to [Testray history](https://testray.liferay.com/home/-/testray/cases/1518352871/view?statuses=3%2C2&testrayCaseId=1518352871&orderByCol=createDate_sortable&orderByType=desc&status=3%2C2&resetCur=false&delta=50), it failed 12 times in the latest 50 runs.

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by jotascript** [comment]
Just started reviewing :)

**Review by jotascript — APPROVED** [review]

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**